src/metadata/metadata_superblock.c

```diff
@@ -302,8 +303,8 @@ struct ocf_pipeline_properties ocf_metadata_load_sb_recov_pipeline_props = {
 	.steps = {
 		OCF_PL_STEP_FOREACH(ocf_metadata_store_segment,
 				ocf_metadata_load_sb_store_segment_args),
-		OCF_PL_STEP_FOREACH(ocf_metadata_load_segment,
-				ocf_metadata_load_sb_recov_load_segment_args),
+		OCF_PL_STEP_ARG_INT(ocf_metadata_load_segment,                                      
+				metadata_segment_sb_config),  
```

Comment:
I don't get what happened here. Anyway, `ocf_metadata_load_superblock_recovery()` is to be removed, so removing it in first commit will allow to avoid unnecessary changes in this code.